packages/grafana-ui/src/themes/default.ts

```diff
@@ -75,6 +75,7 @@ const theme: GrafanaThemeCommons = {
     xxl: '1440px',
   },
   spacing: {
+    base: SPACING_BASE,
```

Comment:
@Clarity-89 added base to the theme type to make it accessible code that uses the theme (really with all the spacings where numbers and not strings, this mixing of strings with 'px' and numbers is so messy)

Reply:
Yeah, I think this is a good workaround 👍